### ISSUE TITLE
Enable Addition of Facility Attributes by Users

### ISSUE THREAD

**Issue by Dhiraj1405** [hotwax/facilities#193]
## What is the motivation for adding/enhancing this feature?
Our system currently lacks the capability for users to add facility attributes, which limits customization and flexibility in facility management. Facility attributes provide additional context and information about each facility, aiding in categorization, filtering, and decision-making processes. To enhance the usability and customization options for facility management, we propose implementing a feature that allows users to add facility attributes according to their specific requirements.

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] 

YES
- [ ] NO

## Additional information
Will add UI in sometime

**Comment by dt2patel** [comment]
What is the exact use case for us to use facility attributes? If you can give particulars @Dhiraj1405 that will help prioritize

**Comment by dt2patel** [comment]
I don't think this requirement if valid since this can just be covered by facility identifications. If we are using facility attributes for this we should migrate to facility identifications.
